### PR TITLE
xlibsWrapper: remove deprecated and now unused wrapper package

### PR DESCRIPTION
**Pull request by trofi**
There should be no reason to use this package as it's a remnant of non-modular X. Chances are you do not want every single library it used to pull in:

      freetype fontconfig xorg.xorgproto xorg.libX11 xorg.libXt
      xorg.libXft xorg.libXext xorg.libSM xorg.libICE

Just pick the ones you really need instead.

`nixpkgs` does not have any users of `xlibsWrapper`.

Closes: https://github.com/NixOS/nixpkgs/issues/194054

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -1626,6 +1626,7 @@ mapAliases ({
   xfceUnstable = xfce4-14; # Added 2019-09-17
   xineLib = xine-lib; # Added 2021-04-27
   xineUI = xine-ui; # Added 2021-04-27
+  xlibsWrapper = throw "'xlibsWrapper' has been replaced by it's constituents"; # Converted to throw 2022-12-27
```
Comment (iblech) [inline pkgs/top-level/aliases.nix:1629]: It should be `its`, not `it's`. Thank you for your working on deprecating this old meta package :-)

Comment (trofi) [reply pkgs/top-level/aliases.nix]: Thank you! Updated `it's` to `its` everywhere.

**Comment by trofi** [comment]
Forgot one last user: `gtk2` on `darwin`. Switching to draft until that is sorted.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by trofi** [comment]
Rebased past `gtk2` fix. Should be ready for review now.

**Comment by SuperSandro2000** [comment]
ofborg is merging the PR into master so this is not strictly necessary.

**Comment by SuperSandro2000** [comment]
Thank you so much trofi! ❤️

**Pull request merged**